### PR TITLE
Update K8s UDS Instructions and general cleanup

### PR DESCRIPTION
**Pull request by JacksonDavenport**
### What does this PR do?

Update the UDS instructions for K8s to clarify some of the phrasing and perform some general docs cleanup.

### Motivation

Clarifying technical details

### Preview

https://docs-staging.datadoghq.com/jack.davenport/k8s_uds/agent/kubernetes/apm/?tab=helm

_Original Page_:
https://docs.datadoghq.com/agent/kubernetes/apm/?tab=unixdomainsocket#setup

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Review by wantsui — APPROVED** [review]
This PR looks fine to me from reading it for the APM setup. 👍  As a future action item, we'll have to add instructions for UDS for each supported tracer.

File: content/en/agent/kubernetes/apm.md [hunk 1]
```diff
@@ -25,120 +25,112 @@ In order to start collecting your application traces you must be [running the Da
...
+      - name: apmsocket
+        mountPath: /var/run/datadog/
+    volumes:
+    - hostPath:
+        path: /var/run/datadog/
+        type: DirectoryOrCreate
+    # (...)
+  ```
+
+This configuration creates a directory on the host and mounts it within the Agent. The Agent then creates and listens on a socket file in that directory with the `DD_APM_RECEIVER_SOCKET` value of `/var/run/datadog/apm.socket`. The Application pods can then similarly mount this volume and write to this same socket.
```
Comment (cswatt) [inline content/en/agent/kubernetes/apm.md:127]:
```suggestion
This configuration creates a directory on the host and mounts it within the Agent. The Agent then creates and listens on a socket file in that directory with the `DD_APM_RECEIVER_SOCKET` value of `/var/run/datadog/apm.socket`. The application pods can then similarly mount this volume and write to this same socket.
```

File: content/en/agent/kubernetes/apm.md [hunk 2]
```diff
@@ -187,84 +175,70 @@ Then apply the new configuration:
...
 
-2. **Configure your application pods in order to communicate with the Datadog Agent**:
-    {{< tabs >}}
-{{% tab "UDP" %}}
-If you are sending traces to the Agent by using UDP (`<IP_ADDRESS>:8126`), either automatically with the [Datadog Admission Controller][1], or manually using the downward API to pull the host IP, the application container needs the `DD_AGENT_HOST` environment variable that points to `status.hostIP`:
+### Configure your application pods in order to communicate with the Datadog Agent
+
+{{< tabs >}}
+{{% tab "IP:Port" %}}
+If you are sending traces to the Agent by using (`<IP_ADDRESS>:8126`) you will supply this IP Address to your application pods either automatically with the [Datadog Admission Controller][1] or manually using the downward API to pull the host IP. The application container needs the `DD_AGENT_HOST` environment variable that points to `status.hostIP`:
```
Comment (cswatt) [inline content/en/agent/kubernetes/apm.md:188]:
```suggestion
If you are sending traces to the Agent by using (`<IP_ADDRESS>:8126`) supply this IP address to your application pods—either automatically with the [Datadog Admission Controller][1], or manually using the downward API to pull the host IP. The application container needs the `DD_AGENT_HOST` environment variable that points to `status.hostIP`:
```

File: content/en/agent/kubernetes/apm.md [hunk 3]
```diff
@@ -187,84 +175,70 @@ Then apply the new configuration:
...
-    {{< tabs >}}
-{{% tab "UDP" %}}
-
-Point your application-level tracers to where the Datadog Agent host is using the environment variable `DD_AGENT_HOST`. Refer to the [language-specific APM instrumentation docs][1] for more examples.
-
-[1]: /tracing/setup/
-{{% /tab %}}
-{{% tab "Unix Domain Socket" %}}
+### Configure your application tracers to emit traces:
+After configuring your Datadog Agent to collect traces and giving your Application Pods the configuration on *where* to send Traces, you will need to install the Datadog Tracer into your applications in order to emit the traces. Once this is done the Tracer should automatically send the traces to the relative `DD_AGENT_HOST` (for `IP:Port`) or `DD_TRACE_AGENT_URL` (for UDS) endpoint.
```
Comment (cswatt) [inline content/en/agent/kubernetes/apm.md:235]:
```suggestion
After configuring your Datadog Agent to collect traces and giving your application pods the configuration on *where* to send traces, install the Datadog Tracer into your applications to emit the traces. Once this is done, the tracer automatically sends the traces to the relative `DD_AGENT_HOST` (for `IP:Port`) or `DD_TRACE_AGENT_URL` (for UDS) endpoint.
```

File: content/en/agent/kubernetes/apm.md [hunk 4]
```diff
@@ -187,84 +175,70 @@ Then apply the new configuration:
...
 
-Point your application-level tracers to where the Datadog Agent host is using the environment variable `DD_APM_RECEIVER_SOCKET` or for Helm `apm.socketPath`. For Helm and Operator the default for this value is `"/var/run/datadog/apm.socket"`. 
-
-Some tracers do not support Unix Domain Socket. Refer to the [language-specific APM instrumentation docs][1] to see if this is supported for your tracer.
+Refer to the [language-specific APM instrumentation docs][1] for more examples.
 
+**Note:** The .NET and PHP Tracers do not currently support sending traces over Unix Domain Socket (UDS).
```
Comment (cswatt) [inline content/en/agent/kubernetes/apm.md:239]:
```suggestion
**Note:** The .NET and PHP tracers do not support sending traces over Unix Domain Socket (UDS).
```

**Review by cswatt — CHANGES_REQUESTED** [review]
style review

**Review by cswatt — APPROVED** [review]

**Pull request merged**